mkosi/__init__.py

```diff
@@ -946,8 +948,7 @@ def build_initrd(state: MkosiState) -> Path:
     ]
 
     with complete_step("Building initrd"):
-        args, presets = parse_config(cmdline)
-        config = presets[0]
+        args, [config] = parse_config(cmdline)
```

Comment:
I'm a bit surprised this type checks, since this would throw a `ValueError` if `presets` had more than one element. Throwing away everything but the first element would be
```suggestion
        args, [config, *_] = parse_config(cmdline)
```
but apparently mypy can prove that there will be only one element?

Reply:
I guess so, I wouldn't complicate things if it type checks, since we know there will only be a single config